SchemaInventory.json

```diff
@@ -4726,6 +4740,20 @@
       "date": "3/1/2023",
       "dynamic": "No",
       "approved": "Yes"
+    },
+    {
+      "name": "IntegratedStructuralModel",
+      "path": "Domains\\4-Application\\ISM\\Released\\IntegratedStructuralModel.01.00.04.ecschema.xml",
+      "released": true,
+      "version": "01.00.04",
+      "comment": "",
+      "verifier": "Diego.Diaz",
+      "sha1": "71a9fde157ab55049ad8889161e8baee1b7a69f7",
+      "verified": "No",
+      "author": "Haroldas.Vitunskas",
+      "date": "4/17/2024",
+      "dynamic": "No",
+      "approved": "No"
```

Comment:
@haroldasvitunskas a released schema must be approved and its value must be set to `Yes` in order to pass the validation.